tests/ci/docker_manifests_merge.py

```diff
@@ -167,6 +173,77 @@ def create_manifest(image: str, tags: List[str], push: bool) -> Tuple[str, str]:
     return manifest, "OK"
 
 
+def enrich_images(changed_images: Dict[str, str]) -> Dict[str, str]:
+    new_changed_images = changed_images
```

Comment:
I've got another idea, that makes sense to implement.

Not sure how good is it though.

If instead of having `new_changed_images` we'd actually append images to existing `changed_images`, and return nothing, we could have partially enriched images.

What's better? Having no images at all, or having a part?